tests/test-definition.yaml

```diff
@@ -85,6 +89,12 @@ tests:
       - nifi-latest
       - zookeeper-latest
       - openshift
+  - name: oidc
+    dimensions:
+      - nifi-latest # it looks like there are compatibility issues with NiFi 2.0.0-M4
```

Comment:
I'm a bit torn here, if we should rather let the tests fail to make it explicit there is still some work to do or should make it pretty clear in the docs that it is not supported for now + create a follow-up task and put it on the daily board

Reply:
I've made changes to the OIDC integration test and now it's working for all of our versions :)

Reply:
Awesome, thanks!